src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -9,7 +9,7 @@
 import java.util.List;
 
 @Data
-//@Builder
+@Builder
```

Comment:
do czego potrzebujemy @Builder ?

Reply:
no właśnie jak dla mnie do niczego :/ ale jest na masterze i cały czas wywala mi tu konflikt

Reply:
https://projectlombok.org/features/Builder

Reply:
i co wyjaśnia ten link? bo nie odpowiada na pytanie do czego w naszym projekcie potrzebujemy @buildera

Reply:
Applying @Builder to a class is as if you added @AllArgsConstructor(access = AccessLevel.PACKAGE) to the class and applied the @Builder annotation to this all-args-constructor.

Reply:
But why we need this in the project? 
If we delete @Builder from the class and leave @Data what happend? What would do not work then?

Reply:
> But why we need this in the project?
> If we delete @builder from the class and leave @DaTa what happend? What would do not work then?

I guess everything will work fine since nobody used that builder pattern in tests. But anyway, here is an example:

Go into your project, to any class that uses @Builder and go : mouse right click -> refractor -> delombok -> @Builder

This will show you the whole method that @Builder provides. It is usefull to construct test data, also builder class handles the initialization of all fields in annotated class.
https://stackoverflow.com/questions/36909672/how-to-use-lombok-builder-annotation-on-methods